package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "channelape-web-service-sdk",
-	"version": "0.1.0",
+	"version": "0.1.0-develop.0",
```

Comment:
Please also merge #4 into this PR so that we do not need to increment the versions twice.